dependencies.props

```diff
@@ -65,7 +66,12 @@
       <ElementName>CoreFxVersion</ElementName>
       <PackageId>Microsoft.Private.CoreFx.NETCoreApp</PackageId>
     </XmlUpdateStep>
-    <XmlUpdateStep Include="CoreFx">
+    <XmlUpdateStep Include="CoreFxUap">
```

Comment:
These should correspond to the `RemoteDependencyBuildInfo` being used--this lets source-build determine which update steps correspond to repos being built. (Items with the same identity are fine.)

Reply:
I'll move this back so it works correctly.  But in general Items with the same identity are not good ideas in MSBuild.  IMO - This should have been a separate metadata property to link them together.

Reply:
I agree, there's a TODO in source-build for this. Was doing what I could with what happened to be in the repos at the time. Dependency flow will change a lot soon so this won't matter--in fact, you could keep the identities you have here since source-build isn't targeting master branches.